src/utils/include/utils/math/vec_rotate.hpp

```diff
@@ -29,26 +29,24 @@ namespace Utils {
 /**
  * @brief Rotate a vector around an axis.
  *
+ * Rodrigues' rotation formula:
+ * @f$ \vec{v}_{\mathrm{rot}} =
+ *      (\cos{\alpha})\vec{v} +
+ *      (\sin{\alpha})\vec{k}\times\vec{v} +
+ *      (1 - \cos{\alpha})(\vec{k}\cdot\vec{v})\vec{k} @f$
+ *
  * @param axis The axis to rotate about
  * @param alpha Angle to rotate
- * @param vector Vector to act on
+ * @param vector %Vector to act on
```

Comment:
is this `%` intended?

Reply:
Yes, otherwise it's interpreted as a `Utils::Vector` and a link is created (https://stackoverflow.com/a/20100255).